.github/workflows/build.yml

```diff
@@ -24,7 +24,6 @@ on:
 jobs:
   build:
     name: Build Wave
-    if: "github.event == 'push' || github.repository != github.event.pull_request.head.repo.full_name"
```

Comment:
Why are we removing this line here too?

Reply:
Come from merging https://github.com/seqeralabs/wave/pull/370, as I needed to run CI tests appropriately within this PR

Reply:
please review https://github.com/seqeralabs/wave/pull/370 first :-)